samples/demo-android/src/main/java/com/apadmi/mockzilla/demo/engine/Repository.kt

```diff
@@ -0,0 +1,49 @@
+package com.apadmi.mockzilla.demo.engine
+
+import com.apadmi.mockzilla.demo.engine.network.DataResult
+import com.apadmi.mockzilla.demo.engine.network.MockzillaTokenProvider
+
+import io.ktor.client.*
+import io.ktor.client.engine.okhttp.OkHttp
+import io.ktor.client.request.*
+import io.ktor.client.statement.*
+import io.ktor.http.*
+import okhttp3.OkHttpClient
+import okhttp3.Protocol
+
+import java.net.Proxy
+
+import kotlinx.serialization.json.Json
+
+class Repository(
```

Comment:
Little thing: could we call this `AnimalRepository`? It looks strange just as `Repository`.

Reply:
Fair enough